build/archive.js

```diff
@@ -229,8 +229,13 @@ function exec(cmdLine, baton, callback) {
     }
 
     child_process.exec(cmdLine, function (err, stdout, stderr) {
-        var err = err || stderr;
-        err && baton ? error(err, baton) : callback((stdout || '').trim(), err);
+        if (!err) { err = stderr; }
+
+        if (err && baton) {
+            error(err, baton);
+        } else {
+            callback((stdout || '').trim(), err);
+        }
```

Comment:
Ah, you know what. I've always been on the fence (and went both ways). Since I tend to use no `{}` in ifs these days, I gravitate towards that, but, I agree with you on the easier readability. I more or less went this way out of haste, haha. So, if you prefer `&&`s, and want to modify the .jshintrc, please do!? (I can always take this commit out, or do the same if you don't get a chance to anytime soon). :-)